ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -278,6 +278,7 @@ service_env:
     - ../../../../ansible/roles/stack-sunbird/templates/transformer_industry_feedback.xml
   orchestrator:
     - ../../../../ansible/roles/stack-sunbird/templates/orchestrator.env
+  fusionauth: ../../../../ansible/roles/stack-sunbird/templates/fusionauth.env
```

Comment:
Why do we need an empty env file?

Reply:
We are using 'sunbird-deploy' job, it is throwing an error, if we don't have env file.

Reply:
We have role named `helm-deploy` which is meant for such charts where we dont need env. Did you try with that role?

Reply:
No, I will check.

Reply:
Tested with helm-deploy, deployment is successfully done. Removed env file and related changes.